support-frontend/assets/pages/weekly-subscription-landing/components/content/productInfo.tsx

```diff
@@ -0,0 +1,148 @@
+import { css } from '@emotion/react';
+import {
+	body,
+	from,
+	headline,
+	palette,
+	space,
+} from '@guardian/source-foundations';
+import GridImage from 'components/gridImage/gridImage';
+import Hero from 'components/page/hero';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import {
+	detect,
+	GBPCountries,
+} from 'helpers/internationalisation/countryGroup';
+import type { PromotionCopy } from 'helpers/productPrice/promotions';
+import { promotionHTML } from 'helpers/productPrice/promotions';
+
+interface ProductInfoProps {
+	promotionCopy: PromotionCopy;
+	orderIsAGift: boolean;
+}
+
+const styles = {
+	heroOverrides: css`
+		background-color: #cadbe8;
+	`,
+	showOnMobile: css`
+		display: block;
+
+		${from.mobileLandscape} {
+			display: none;
+		}
+	`,
+	productCopy: css`
+		padding: 0 ${space[3]}px ${space[3]}px;
+		color: ${palette.neutral[7]};
+	`,
+	productTitle: css`
+		${headline.small({
+			fontWeight: 'bold',
+		})};
+		margin-bottom: ${space[3]}px;
+
+		${from.mobileLandscape} {
+			width: 75%;
+		}
+
+		${from.tablet} {
+			${headline.large({
+				fontWeight: 'bold',
+			})};
+			width: 100%;
+		}
+	`,
+	productParagraph: css`
+		${body.medium({
+			lineHeight: 'loose',
+		})}
+		margin-bottom: ${space[9]}px;
+
+		/* apply the same margin to paragraphs parsed from markdown from promo codes */
+		& p:not(:last-of-type) {
+			margin-bottom: ${space[9]}px;
+		}
+	`,
+};
+
+const getRegionalCopyFor = (region: CountryGroupId) =>
+	region === GBPCountries ? (
+		<span>
+			Find clarity
+			<br css={styles.showOnMobile} /> with the Guardian&apos;s global magazine
+		</span>
+	) : (
+		<span>
+			Read The
+			<br css={styles.showOnMobile} /> Guardian in print
+		</span>
+	);
+
+const getFirstParagraph = (
+	promotionCopy: PromotionCopy,
+	orderIsAGift: boolean,
+) => {
+	if (promotionCopy.description) {
+		return promotionHTML(promotionCopy.description);
+	}
+
+	if (orderIsAGift) {
+		return (
+			<>
+				<p>
+					Gift the Guardian Weekly magazine to someone today, so they can gain a
+					deeper understanding of the issues they care about. They’ll find
+					in-depth reporting, alongside news, opinion pieces and long reads from
+					around the globe. From unpicking the election results to debunking
+					climate misinformation, they can take time with the Guardian Weekly to
+					help them make sense of the world.
```

Comment:
I’m surprised by this added text (here and below) – I was expecting it would be replacing something, or moved here from somewhere, but I don’t see any corresponding deletions. Is this text just used for the variant? (I couldn’t see it in the figma designs either, but I may be looking in the wrong place.)

Reply:
This is a really good spot! There was a bit of moving things around with this one but instead of creating these new functions I should be exporting and reusing the existing functions in [`hero.tsx`](https://github.com/guardian/support-frontend/blob/da32adb7eff4cdb089a7e9af7fe1068f1d2c8080/support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.tsx#L96) here

Reply:
This is definitely the variant side Emily, and can see the copy in Figma (blue 'Open Up Your World...')